x/auction/types/params_test.go

```diff
@@ -1,33 +1,101 @@
 package types
 
 import (
-	"github.com/stretchr/testify/require"
 	"testing"
 	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
 )
 
 func TestParams_Validate(t *testing.T) {
 	type fields struct {
 	}
 	testCases := []struct {
-		name               string
-		MaxAuctionDuration time.Duration
-		BidDuration        time.Duration
-		expectErr          bool
+		name string
+		Params
+		expectErr bool
 	}{
-		{"normal", 24 * time.Hour, 1 * time.Hour, false},
-		{"negativeBid", 24 * time.Hour, -1 * time.Hour, true},
-		{"negativeAuction", -24 * time.Hour, 1 * time.Hour, true},
-		{"bid>auction", 1 * time.Hour, 24 * time.Hour, true},
-		{"zeros", 0, 0, false},
+		{
+			"normal",
+			DefaultParams(),
+			false,
+		},
+		{
+			"negativeBid",
+			Params{
+				MaxAuctionDuration:  24 * time.Hour,
+				BidDuration:         -1 * time.Hour,
+				IncrementSurplus:    d("0.05"),
+				IncrementDebt:       d("0.05"),
+				IncrementCollateral: d("0.05"),
+			},
+			true,
+		},
+		{
+			"negativeAuction",
+			Params{
+				MaxAuctionDuration:  -24 * time.Hour,
+				BidDuration:         1 * time.Hour,
+				IncrementSurplus:    d("0.05"),
+				IncrementDebt:       d("0.05"),
+				IncrementCollateral: d("0.05"),
+			},
+			true,
+		},
+		{
+			"bid>auction",
+			Params{
+				MaxAuctionDuration:  1 * time.Hour,
+				BidDuration:         24 * time.Hour,
+				IncrementSurplus:    d("0.05"),
+				IncrementDebt:       d("0.05"),
+				IncrementCollateral: d("0.05"),
+			},
+			true,
+		},
+		{
+			"negative increment surplus",
```

Comment:
These tests are hitting error "bid duration param cannot be larger than max auction duration" and not the expected errors.

Reply:
good catch